### ISSUE TITLE
Incorrect handling of multi-line ExecStart directives

### ISSUE THREAD

**Issue by ZetaTwo** [desbma/shh#2]
Systemd directives can span multiple lines by endling the line with a backslash. For example, MariaDB Debian systemd unit contains this directive:
```
ExecStartPre=/bin/sh -c "[ ! -e /usr/bin/galera_recovery ] && VAR= || \
 VAR=`cd /usr/bin/..; /usr/bin/galera_recovery`; [ $? -eq 0 ] \
 && systemctl set-environment _WSREP_START_POSITION=$VAR || exit 1"
```

when this is processed in service.rs:114-132 it creates the following result:

```
ExecStartPre=/usr/local/bin/shh run -m safe -p /run/shh-profile-data_f148f50a/003 -- /bin/sh -c "[ ! -e /usr/bin/galera_recovery ] && VAR= || \
```

the expected result is something like:
```
ExecStartPre=/usr/local/bin/shh run -m safe -p /run/shh-profile-data_f148f50a/003 -- /bin/sh -c "[ ! -e /usr/bin/galera_recovery ] && VAR= || \
 VAR=`cd /usr/bin/..; /usr/bin/galera_recovery`; [ $? -eq 0 ] \
 && systemctl set-environment _WSREP_START_POSITION=$VAR || exit 1"
```

**Comment by desbma** [comment]
Yeah, the systemd service configuration parser is probably currently too simplistic, I'll look at it next week.

**Comment by desbma** [comment]
It should be fixed by 91d363c1a5d66241daa4297ae46c88cdb138483a
